documentation.css

```diff
@@ -177,6 +196,56 @@ body {
 }
 
 .doc-content {
-    width: 80%;
     margin-left: 2rem;
+}
+/*----------------------------- Responsiveness-----------------------------------  */
+@media only screen and (max-width:1000px) {
+
+    .universal-container {
+
+        display: grid;
+        height: 100%;
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+        grid-template-areas:
+            "aside-nav  main main main main ";
+    }
+
```

Comment:
Add come padding on the right for this container. Rest everything looks good.

Reply:
Done with this too . Thanks for pointing this out :)